spec/models/activerecord_obstacle_course_spec.rb

```diff
@@ -349,7 +349,7 @@
     Bullet.start_request
 
     # ------------------------------------------------------
-    orders = Order.all # Edit only this line
+    orders = Order.includes(:items) # Edit only this line
```

Comment:
This goes over it a little bit: https://www.sitepoint.com/silver-bullet-n1-problem/

In short, the idea is that using `.includes` loads the information for `items` into the local variable `orders`. By doing this, it puts that information *into memory*. That way when ruby makes an `.each` call on the local variable `orders`, and then finds information about the `items` related to that order, your machine doesn't have to go back to the database each time through the loop (that information has already been loaded into the variable `orders`, which is why this is called *eager loading*). You're loading information you know you'll need, even if you don't use it right away.